### PR TITLE
Update Copyright Year & Wallet Image

### PR DESCRIPTION
**Pull request by JaredTate**
Needed to update the copyright splash image to 2023 and throughout the client. Also needed to remove "test" from the wallet image.

### CODE REVIEW

**Review by ycagel — DISMISSED** [review]
cACK - Looks straightforward to me.

File: src/qt/res/icons/digibyte_wallet.png
Comment (gto90) [inline src/qt/res/icons/digibyte_wallet.png:1]: @JaredTate , could we get the version to read: v8.22.0-RC1 or similar?

**Review by ycagel — APPROVED** [review]
cACK - Good update.

**Review by gto90 — APPROVED** [review]
cACK

**Review by SmartArray — APPROVED** [review]
utACK

**Comment by saltedlolly** [comment]
Question - if this is released as is - i.e. without an algo swap to RandomX/ProgPOW - are there any included changes that require a hard fork?

**Comment by saltedlolly** [comment]
Also, could someone with access update the URL for the repo from digibyte.io to digibyte.org

**Comment by ycagel** [comment]
@saltedlolly Updated to DigiByte.org

**Pull request merged**